.github/workflows/downstream_quickstarts.yml

```diff
@@ -0,0 +1,54 @@
+name: Downstream - Timefold Quickstarts
+
+on:
+  pull_request:
+    branches: [main]
```

Comment:
Should we support `.x` branches?
How should it behave then?

Reply:
It does not hurt setting it up already. I think it should already work with you script for PRs over both repositories. For a PR to solver only, you have to provide the right default branch (the same `.x` branch) from the target organization and repository.